android/app/src/main/python/mmrelay/cli.py

```
@@ -0,0 +1,1614 @@
+                    from mmrelay.e2ee_utils import (
+                        get_e2ee_status,
+                    )
+
+                    e2ee_status = get_e2ee_status(config, config_path)
+                    _print_unified_e2ee_analysis(e2ee_status)
+
+                    # Check if there are critical E2EE issues
+                    if not e2ee_status.get("platform_supported", True):
+                        print("\n⚠️  Warning: E2EE is not supported on Windows")
+                        print("   Messages to encrypted rooms will be blocked")
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)